src/renderer/components/pages/Config/pages/ArmPresetsConfig/ArmJointInput.tsx

```diff
@@ -40,12 +41,12 @@ const ArmJointInput = ({
         value={inputValue}
         type="number"
         onChange={onInputChange}
+        labelStyle={
+          error
+            ? { color: defaultTheme.colors.danger }
+            : { color: defaultTheme.colors.fontLight }
+        }
       />
```

Comment:
Keep the validation message!

Reply:
I removed it, because it looked a bit weird with the new UI, but more info is more important than looks so I added it back :)